frontend/src/components/Sheet/Sheet.module.scss

```diff
@@ -25,6 +25,34 @@
   color: $secondary-yellow;
 }
 
+.containerDropdownNavbar {
+  z-index: 3;
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  width: 100vw;
+  height: calc(100vh - 60px);
```

Comment:
nit but try to avoid stuff like this because if we change the height of the nav will have to change this everywhere (could make a scss variable or something)

Reply:
tbh you shouldn't need to do this though - things should be set up that 100% height fills from the bottom of the nav to the bottom of the page

Reply:
made a scss var